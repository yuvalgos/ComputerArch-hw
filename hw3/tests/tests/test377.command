./dflow_calc tests/test377.dat tests/test377.in d33 d149 d184 p13 p166 p136 d96 d13 p103 p150 p139 p100 d14 d194 p78 d66 p83 p154 d62 d21 p133 p115 p141 p129 p55 p180 d79 d135 d41 p44 d166 p118 d9 d55 d36 d163 p181 d80 d163 p146 p154 d61 d78 p130 p21 p0 p35 p144 d144 d152 p151 d59 d151 d96 d107 p167 d16 p132 p85 p57 p78 d180 p38 d141 p148 d148 d3