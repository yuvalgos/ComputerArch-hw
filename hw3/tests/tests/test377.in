3 7 4 7
8 19 20 24
7 6 24 8
5 30 25 3
4 21 8 4
8 23 0 4
4 10 8 5
5 7 18 6
7 19 17 0
3 6 31 15
4 30 15 30
8 13 7 13
8 21 18 23
2 25 6 12
0 4 25 9
7 17 5 10
4 0 18 30
2 26 25 15
5 4 26 30
1 29 23 0
4 31 26 21
0 5 26 30
4 18 26 2
5 19 25 15
1 19 0 6
0 13 20 12
0 17 4 3
3 30 10 28
6 27 12 14
8 29 19 2
2 30 11 8
2 1 27 2
3 31 22 29
7 3 20 29
6 27 25 26
6 9 13 20
2 23 27 7
8 24 14 5
1 7 30 20
3 13 9 31
5 5 6 3
8 14 24 3
6 18 4 22
6 13 24 5
0 4 22 7
4 16 19 16
1 30 15 1
8 2 28 13
2 4 24 6
5 11 29 19
2 23 5 11
6 22 0 8
0 25 21 12
8 18 17 28
4 0 4 1
0 25 10 11
0 6 17 31
0 11 0 20
2 23 9 15
8 6 24 26
4 20 20 19
4 6 27 11
6 7 0 11
8 29 12 10
1 2 4 16
4 2 15 19
8 20 13 25
1 16 10 4
0 0 29 14
7 8 18 7
4 9 16 5
2 26 14 21
1 19 15 6
1 28 9 7
1 6 22 26
8 29 31 3
2 2 23 12
3 23 18 16
2 31 28 20
3 6 17 14
1 31 19 13
6 4 9 16
3 31 6 6
6 25 0 4
4 5 14 8
8 1 23 8
5 25 7 20
1 22 23 14
6 12 24 18
5 12 13 1
6 8 31 7
7 4 0 27
6 31 22 22
2 2 31 2
2 22 29 21
7 12 16 18
8 0 23 31
7 0 7 13
8 18 2 8
6 9 23 13
3 26 25 17
2 3 9 13
7 12 23 3
5 26 12 16
1 13 0 8
8 25 4 30
0 25 17 25
0 10 8 21
2 25 0 9
3 15 27 16
1 19 11 28
7 20 1 24
1 4 13 13
7 6 3 16
0 23 22 30
1 4 1 29
8 16 15 13
6 31 2 29
7 27 0 11
4 1 4 26
2 2 21 23
7 20 25 12
7 31 31 3
4 21 2 12
0 5 19 29
5 20 1 21
7 18 26 29
3 10 22 12
5 21 15 30
3 0 14 17
5 30 21 10
8 11 29 17
4 14 3 8
3 28 9 24
8 8 5 8
5 0 31 10
6 18 14 9
6 12 1 23
8 11 1 26
0 17 22 19
2 13 26 11
0 15 23 1
6 22 11 18
4 30 12 18
2 8 20 21
6 8 2 27
3 25 23 17
4 31 7 21
3 7 16 5
0 6 3 25
4 4 4 4
4 11 3 6
6 28 23 18
3 17 7 14
1 29 19 18
1 0 21 18
5 9 8 17
8 11 27 24
6 10 14 5
5 17 30 0
2 30 8 12
3 17 4 30
6 4 5 8
6 28 9 13
3 21 17 14
6 11 14 0
0 1 22 24
2 18 22 7
8 31 3 4
6 22 13 7
7 0 28 8
6 0 25 14
5 5 0 14
2 19 10 16
3 4 20 14
4 9 21 17
5 15 26 1
4 24 21 23
7 3 0 5
0 29 11 22
1 2 12 31
2 29 0 3
7 2 24 30
0 0 2 24
3 9 6 13
2 6 16 28
3 3 10 7
6 10 21 17
1 23 30 22
0 17 27 5
6 30 4 5
1 3 8 0
2 26 4 30
2 5 20 9
2 11 27 28
8 31 8 28
5 26 1 30
4 8 29 9
4 22 31 18
0 30 4 2
