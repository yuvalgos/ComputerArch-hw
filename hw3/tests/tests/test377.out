Reading the opcodes latency info from tests/test377.dat ... Got latency for 9 opcodes
Reading the program file tests/test377.in ... Found 200 instructions
getProgDepth()==254
getInstDeps(33)=={-1,29}
getInstDeps(149)=={101,146}
getInstDeps(184)=={149,140}
getDepDepth(13)==7
getDepDepth(166)==215
getDepDepth(136)==145
getInstDeps(96)=={77,92}
getInstDeps(13)=={9,-1}
getDepDepth(103)==151
getDepDepth(150)==129
getDepDepth(139)==110
getDepDepth(100)==146
getInstDeps(14)=={13,-1}
getInstDeps(194)=={118,163}
getDepDepth(78)==133
getInstDeps(66)=={43,55}
getDepDepth(83)==121
getDepDepth(154)==162
getInstDeps(62)=={54,57}
getInstDeps(21)=={17,10}
getDepDepth(133)==127
getDepDepth(115)==126
getDepDepth(141)==175
getDepDepth(129)==103
getDepDepth(55)==74
getDepDepth(180)==205
getInstDeps(79)=={26,41}
getInstDeps(135)=={122,127}
getInstDeps(41)=={37,33}
getDepDepth(44)==8
getInstDeps(166)=={142,37}
getDepDepth(118)==129
getInstDeps(9)=={-1,-1}
getInstDeps(55)=={6,49}
getInstDeps(36)=={34,7}
getInstDeps(163)=={156,140}
getDepDepth(181)==194
getInstDeps(80)=={72,43}
getInstDeps(163)=={156,140}
getDepDepth(146)==112
getDepDepth(154)==162
getInstDeps(61)=={34,57}
getInstDeps(78)=={73,66}
getDepDepth(130)==158
getDepDepth(21)==24
getDepDepth(0)==0
getDepDepth(35)==2
getDepDepth(144)==188
getInstDeps(144)=={125,128}
getInstDeps(152)=={114,136}
getDepDepth(151)==134
getInstDeps(59)=={37,17}
getInstDeps(151)=={101,149}
getInstDeps(96)=={77,92}
getInstDeps(107)=={90,12}
getDepDepth(167)==215
getInstDeps(16)=={-1,10}
getDepDepth(132)==134
getDepDepth(85)==103
getDepDepth(57)==65
getDepDepth(78)==133
getInstDeps(180)=={137,168}
getDepDepth(38)==5
getInstDeps(141)=={114,119}
getDepDepth(148)==160
getInstDeps(148)=={116,124}
getInstDeps(3)=={-1,-1}
